addons/interact_menu/functions/fnc_collectActiveActionTree.sqf

```diff
@@ -48,7 +48,7 @@ _fullPath pushBack _actionName;
 private _activeChildren = [];
 
 // If there's a statement to dynamically insert children then execute it
-if !({} isEqualTo _insertChildrenCode) then {
+if ({} isNotEqualTo _insertChildrenCode) then {
```

Comment:
Can anyone shed some light on why this fails in SQF validator?

Reply:
because of this: https://github.com/acemod/ACE3/blob/master/tools/sqf_validator.py#L11
isNotEqualTo is not a valid command after Code. just add it in the array and the issue should be solved

Reply:
Just reverse it to
```
_insertChildrenCode isNotEqualTo {}
``´

Makes more sense anyway.

Reply:
this should still be added to the validator!